addons/stock/wizard/stock_replenishment_info.xml

```diff
@@ -7,13 +7,13 @@
             <form>
                 <field name="orderpoint_id" invisible="1"/>
                 <field name="qty_to_order" invisible="1"/>
-                <div class="row ml8">
-                    <group class="col-6">
-                        <field nolabel="1" name="json_lead_days" widget="lead_days_widget"/>
-                    </group>
-                    <group class="col-6">
-                        <field nolabel="1" name="json_replenishment_history" widget="replenishment_history_widget"/>
-                    </group>
+                <div class="row">
+                    <div class="col-md-6 col-sm-8">
+                        <field name="json_lead_days" widget="lead_days_widget"/>
+                    </div>
+                    <div class="col-md-6 col-sm-4">
+                        <field class="d-block" name="json_replenishment_history" widget="replenishment_history_widget"/>
+                    </div>
```

Comment:
hello @AhmedAmirK 

we're about to send some guidelines, but in the mean time, don't apply this diff but take this suggestion indeed
```suggestion
                <div class="row ml8">
                    <group>
                        <group>
                            <field nolabel="1" colspan="2" name="json_lead_days" widget="lead_days_widget"/>
                        </group>
                        <group>
                            <field nolabel="1" colspan="2" name="json_replenishment_history" widget="replenishment_history_widget"/>
                        </group>
                    </group>
                </div>

```

You just need to define colspan values inside of a grid, it should be more maintainable
![image](https://user-images.githubusercontent.com/9530767/194285954-3b02a62a-772f-43b5-aee9-d394b7200587.png)

_NOTE_: right now, the widgets don't take the whole width of their columns, this is about to be fixed